src/helm/blue-agent/values.yaml

```diff
@@ -126,6 +126,9 @@ tolerations: []
 # Pod affinity configuration
 affinity: {}
 
+# Number of revisions to keep in the history of the Deployment
```

Comment:
Value to `0` will cause the inability to rollback. Can you specify the documentation link as note plus a warning?

https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#clean-up-policy